src/persist-txn/src/txns.rs

```diff
@@ -251,7 +251,7 @@ where
     /// for an unbounded amount of time.
     ///
     /// This method is idempotent.
-    pub async fn apply_le(&mut self, ts: &T) {
+    pub async fn apply_le(&mut self, ts: &T) -> Tidy {
```

Comment:
The doc-comment currently says

> /// In the common case, the txn committer will have done this work and this
> /// method will be a no-op, but it is not guaranteed. In the event of a
> /// crash or race, this does whatever persist writes and resulting
> /// maintenance work is necessary, which could be significant.

It should probably remove the part about doing the "resulting maintenance work is necessary" and instead say that it will return the "resulting maintenance work is necessary".

Reply:
Done!